docs/reference/cli.md

```diff
@@ -228,6 +228,35 @@ pixi run --clean-env "echo \$PATH"
         pixi run start
         ```
 
+## `exec`
+
+Runs a command in a temporary environment disconnected from any project.
+This can be useful to quickly test out a certain package or version.
+
+Temporary environments are cached. If the same command is run again, the same environment will be reused.
```

Comment:
Is there a way to find this cache, e.g. for cleaning?

Reply:
I added some information about where to find the environments and that they should be deleted manually.